src/main/java/uk/gov/hmcts/reform/managecase/client/prd/ProfessionalUser.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.managecase.client.prd;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+@Builder
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ProfessionalUser {
+
+    private String userIdentifier;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String idamStatus;
+    private List<String> roles;
+    private String organisationIdentifier;
```

Comment:
this wont be returned, actually.

Reply:
The organisation identifier will be return at top level of 'FindUsersByOrganisationResponse'.

Reply:
ohh.. is that confirmed?

Reply:
done